Placements entry added (#329)
DFE-Digital/bat-design-history: app/posts/register-trainee-teachers/2021-02-22-collecting-placement-details.md

>
>* a four-year undergraduate programme - 160 days (32 weeks)
>* a one, two or three-year undergraduate programme - 120 days (24 weeks)
>* a secondary graduate (non-employment-based) programme - 120 days (24 weeks)
>* a primary graduate (non-employment-based) programme - 120 days (24 weeks)
>* employment-based programme - as determined by the training programme”

Source: [Initial teacher training (ITT): criteria and supporting advice](https://www.gov.uk/government/publications/initial-teacher-training-criteria/initial-teacher-training-itt-criteria-and-supporting-advice#c23-training-in-schools)

However, the pre-COVID guidance is far more specific about the number of schools that trainee must have taught in:

>“Trainees must have taught in 2 schools prior to the award of QTS. It is not sufficient for trainees only to have had experience of working in 2 schools. Providers may wish to consider whether a trainee with prior assessed experience of teaching in a school has gained enough relevant teaching experience to allow the provider confidently to count that previous teaching as one of the 2 schools specified.”

> We do not know what the guidance will be in the next academic year. This will have a significant impact on how  we collect placement details.

Reviewer: ```suggestion
We do not know what the guidance will be in the next academic year.
```

## Designs

### When adding a new trainee

{{ appFigure({
  image: {
    file: "adding-a-new-trainee.png",
    alt: "Adding a new trainee"
  },
  caption: "‘Placement details’ has been added to the ‘About their Training’ section on the draft overview screen for all routes except assessment only."
}) }}

#### Does the user have placement details yet?

When do users know placement details? It differs. We need to take 3 potential scenarios into consideration at the time a draft is created:

* all placements details are known
* some placement details are known
* no placement details are known

Answering ‘No, I’ll add them later’ marks the section complete, while answering ‘Yes, I can add at least one of them now’ takes the user to the ‘Placement details’ section.

{{ appFigure({
  image: {
    file: "does-the-user-have-placement-details-yet.png",

>     alt: "Do you have the trainee's placement details?"

Reviewer: ```suggestion
    alt: "Do you have the trainee’s placement details?"
```

  }
}) }}

#### Adding a placement

We ask for the placement school’s unique reference number (URN) in a plain text field. We assume there are many schools with similar names so we need a unique identifier to tell them apart. 

The next iteration on this field will most likely be a search with autocomplete results.

> In DTTP, placement duration is recorded in total number of days. We’re assuming, however, that some users might not have start and finish dates or know the exact number of days at the time of inputting the data.  We also don’t think it’s clear whether users should only count weekdays or full weeks.

Reviewer: ```suggestion
In DTTP, placement duration is recorded in total number of days. We’re assuming, however, that some users might not have start and finish dates or know the exact number of days at the time of inputting the data. We also don’t think it’s clear whether users should only count weekdays or full weeks.
```

    alt: "Trainee record: confirm placements"
  }
}) }}

#### Have all placements been added?

Once 2 or more placements have been added, the ‘Confirm placement details’ page changes state. The user can now confirm that all placements have been added by checking the ‘Have all placements been added?’ input.

{{ appFigure({
  image: {
    file: "confirm-placement-details-minimum-added.png",
    alt: "Confirm placement details"
  }
}) }}

#### Ready to recommend for QTS

Once the minimum number of placements has been added and the user has confirmed that all placements have been added the trainee can now be recommended for QTS.   

{{ appFigure({
  image: {
    file: "trainee-record-ready-to-recommend.png",
    alt: "Trainee record: ready to recommend"
  }
}) }}

## Testing these designs

Providing placement details has never been mandatory in the process of recommending a trainee for QTS. We’re curious to see how our users respond to this change in process.

Will our users notice the banner prompting for placement details? We hope to validate this.

We also want to understand more about the details providers keep, for example:

* do they know the placement location’s URN?

> * do they know the start and end dates for each placement?  

Reviewer: ```suggestion
* do they know the start and end dates for each placement?
```
